py3-numpy.yaml

```diff
@@ -65,6 +64,8 @@ subpackages:
         - py${{range.key}}-${{vars.pypi-package}}
       provides:
         - py3-${{vars.pypi-package}}
+        - numpy-bin
+        - f2py
```

Comment:
Not your doing, but would you mind adding a basic test?

Reply:
I am suspecting that py3-${{vars.pypi-package}} provides in this py-version range for name: ..-bin is wrong, and should be `py3-${{vars.pypi-package}}-bin`, such that scipy can depend on `py3-numpy-bin`.

The difference is subtle, but it will be possible to copy & paste that to any other -bin packages, instead of introducing bespoke numpy-bin.

the `numpy` & `f2py` provides are for historic/backwards compat, and indeed f2py should be moved here.

Reply:
@xnox right, pushing some updates to do that